src/main/resources/templates/db_scripts/schema.json

```diff
@@ -97,12 +74,6 @@
         }
       ],
       "ginIndex": [
-        {
-          "fieldName": "itemId",
-          "tOps": "ADD",
```

Comment:
Do not remove the lines, change `"tOps": "ADD"` to `"tOps": "DELETE"` so that the index gets removed.
This schema.json doesn't contain "itemId" as "fieldName" in any "foreignKey" section. Therefore an index is needed for searching. Please create an "index" section and add this.